### DISCUSSION TITLE
OKD high CPU usage

### DISCUSSION THREAD

**Discussion by pzi123** [okd-project/okd#1557] [General]
Playing with OKD single node CRC latest/2.16 (OKD 4.12) the and observe about 60% CPU usage with no application deployments installed. I have 4x2.3GHz CPUs assigned to that vSphere VM with 32GB of memory. This single VM puts constant 5GHz load on the ESX 7.0.3 hypervisor. Can anyone explain where the load is coming from?

With that single VM this is not a big problem. I installed standard vSphere OKD 4.9 that deploys 3 master and 3 worker nodes VMs and I see some 20GHz constant load on the hypervisor otherwise completely idle (marginal CPU load). Now this is a problem - I measure power usage of my dual Xeon E5 with 24 cores and the power consumption grew by some 100W.  If running an OpenShift platform with no application load generates 20GHz how can production environments justify using it?

**Comment by vrutkovs** [comment]
>Can anyone explain where the load is coming from?

Check Prometheus metrics. Most likely its etcd+kube-apiserver+Prometheus

**Reply by pzi123** [reply]
Thanks vrutkovs for the monitoring pointer.  Let me change the question a bit:  Are the rest of OKD users here observe that kind of CPU load with OKD idling with no Apps running?

**Reply by samuel-emrys** [reply]
Confirming I'm seeing similar jumps in power draw. Turning on my OKD cluster (3cp 2w) triggers a 160W jump in power consumption in a default install with no workload.
